odoo/addons/test_main_flows/static/tests/tours/main_flow.js

```diff
@@ -890,11 +890,6 @@ tour.stepUtils.mobileModifier(tour.stepUtils.autoExpandMoreButtons('.o_control_p
     edition: "enterprise",
     trigger: "div.o_account_kanban div.o_kanban_card_header a.oe_kanban_action span:contains('Bank')",
     content: _t("Open the bank reconciliation widget"),
-}, {
-    mobile: false,
-    edition: "enterprise",
-    trigger: "button.o_switch_view.o_list",
-    content: _t("Move to the list view"),
```

Comment:
I don't understand what is wrong with that step. Creating from list or kanban is the same, right?

Reply:
The order of the views are changed, clicking on journal name opens the tree view, so this step is not necessary and of course there is no `o_switch_view.o_list` to click on